b2sdk/exception.py

```diff
@@ -397,8 +400,9 @@ class UnrecognizedBucketType(B2Error):
     pass
 
 
-def interpret_b2_error(status, code, message, post_params=None):
+def interpret_b2_error(status, code, message, post_params=None, response_headers=None):
```

Comment:
there is only one caller of `interpret_b2_error` and it's in our internal code, non-public interface. Therefore please make `response_headers` a mandatory argument (between `message` and `post_params` and remove `hasattr(e, 'response_headers')` from `_translate_and_retry()`